event/microserviceEvent.go

```diff
@@ -159,6 +160,26 @@ func (LegendMissionsOngoingMissionEventPayload) Type() MicroserviceEvent {
 	return LegendMissionsOngoingMissionEvent
 }
 
+type CryptoRankingWinners struct {
+	UserID string `json:"userId"`
+	// float64 because of the number of decimals it can have
+	Reward float64 `json:"reward"`
```

Comment:
The reward unit is linked to the wallet in 'legend-blockchain'. The justification for using float64 is because the prize can be as small as 0.000109 BTC

Reply:
>  This also raises another question: what is the unit of the reward (e.g., dollars, USDT, MATIC)?

How does the microservice receiving the event determine the unit of the reward (e.g., dollars, USDT, MATIC)?

Reply:
'legend-rankings' emits the event with the wallet to transfer and its winners. This event will be listened to by 'legend-blockchain'. This microservice stores the wallets generated and linked for the rankings. The wallet will be searched using the 'public_address' field, which will provide information about the network and the unit of the reward

Reply:
ok